Hometask4.playground/Contents.swift

```diff
@@ -0,0 +1,84 @@
+//Создать словарь, где ключ - название города/страны, значение - интересный факт и континент, где распорожен город/страна (подумайте, какой тип данных лучше использовать для значения).
+
+import Foundation
+
+var worldPlaceDictionary = ["Russia": ("The biggest country in the world", "Eurasia"),
+                            "Portugal": ("One of the most safety country in the world","Europe"),
+                            "Australia": ("The same as Europe but without migrants from near East","Australia"),
+                            "Netherlands": ("Gaaga", "Europe"),
+                            "Belarus": ("Misnk is one of the most clean city", "Europe"),
+                            "Taiwan": ("The main center of microconductor manufactor", "Asia"),
+                            "Japan": ("Neutral country", "Asia"),
+                            "USA": ("Top 10 companies with the biggest capitalisation placed in USA", "North America")]
+
+//1. Добавить новый элемент в словарь, поменять два существующих элемента в словаре, удалить один элемент из словаря.
+
+worldPlaceDictionary["Minsk"] = ("1067 battle on Nemiga", "Europe") // add a new element
+
+worldPlaceDictionary["Russia"]    = ("Putinka", "Eurasia")
+worldPlaceDictionary["Australia"] = ("A1", "Australia") // change elements in dictionary
+
+worldPlaceDictionary.removeValue(forKey: "Minsk")
+worldPlaceDictionary["Netherlands"] = nil // remove value from the Dictionary
+
+//2. Создать отдельный словарь с названием городов/стран с длиной более 6 букв из первоначального словаря.
+
+var dictCitiesLongName = [String: (String, String)] ()
+
+dictCitiesLongName = worldPlaceDictionary.filter({ $0.key.count > 6 })
```

Comment:
You can combine these lines of code into one. No need to declare a variable on one line and assign value in another

Reply:
combined